remake.yml

```diff
@@ -0,0 +1,17 @@
+sources:
+	- code.R
+[D[D[D[D[D[D[D[D[A[C[C[C[C[C[C[C[C[C[D[A[C[C[C[C[Csources:
```

Comment:
Not sure why this added the [D...but I think it was the `cat` I asked you to do, which didn't work so well on windows? 

Anyhow, you fixed it